package.json

```diff
@@ -24,7 +26,7 @@
     "prisma": "^5.7.1",
     "react": "^18",
     "react-dom": "^18",
-    "react-hook-form": "^7.50.0",
+    "react-hook-form": "^7.50.1",
```

Comment:
Why downgrade the react hook form package version? 🤔

Reply:
Its update